splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1707,12 +1707,12 @@
         },
         "name": {
           "type": "string",
-          "pattern": "^[^<>:\"\\/\\\\|\\?\\*]+$",
+          "pattern": "^(?!^(CON|PRN|AUX|NUL|COM[1-9]|LPT[1-9])$)(?!.*\\.(tar(\\.gz)?|tgz|spl)$)[A-Za-z_.-][A-Za-z0-9_.-]*[A-Za-z0-9_-]$",
```

Comment:
That would not break any existing TAs from building?

Reply:
This wouldn’t be an issue since we are following Splunk's naming conventions, so any TA name which doesn't follow this naming convention won't be deployable on Splunk.